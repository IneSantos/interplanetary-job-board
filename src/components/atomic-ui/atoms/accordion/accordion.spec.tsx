import React from "react";
import { render } from '@testing-library/react';
import Accordion from "./accordion";

describe("Accordion atom tests", () => {
  test(`Accordion component render`, () => {
    const { container } = render(
      <Accordion id={"accordion1"} label="accordion1" content={null} />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
