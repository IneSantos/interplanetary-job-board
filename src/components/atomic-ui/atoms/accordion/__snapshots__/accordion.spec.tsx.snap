// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion atom tests Accordion component render 1`] = `
<div
  class="accordion"
>
  <div>
    <input
      class="accordion-input"
      id="accordion1"
      type="checkbox"
    />
    <label
      class="accordion-label"
      for="accordion1"
    >
      accordion1
      <i
        class="fas fa-chevron-down"
      />
    </label>
    <div
      class="accordion-content"
    />
  </div>
</div>
`;
